.standard.yml

```diff
@@ -1 +1,4 @@
 parallel: true
+format: progress
+plugins:
+  - standard-rails
```

Comment:
Here's how to pull in the `standard-rails` style guide.

Reply:
add a comment?   also interesting we don't need ones for performance or custom plugins

Reply:
Thinking about Sandi Metz's advice about not adding comments where code already tells the story... the YAML here tells us this is how to load a `standard` plugin such as `standard-rails`, so what would the comment say?